bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/internal/workbench/swt/OnboardingCTabFolder.java

```diff
@@ -0,0 +1,55 @@
+package org.eclipse.e4.ui.internal.workbench.swt;
```

Comment:
Please add a copyright header (copy and adapt from other files)

Reply:
Done